mne/annotations.py

```diff
@@ -977,3 +1024,58 @@ def events_from_annotations(raw, event_id="auto",
                 (list(event_id_.keys()),))
 
     return events, event_id_
+
+
+@verbose
+def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
+                            orig_time=None, verbose=None):
+    """Convert an event array to an Annotations object.
+
+    Parameters
+    ----------
+    events : ndarray, shape (n_events, 3)
+        The events.
+    sfreq : float
+        Sampling frequency.
+    event_desc : dict | callable | None
+        Events description. Can be:
+
+        - **dict**: map integer event codes (keys) to descriptions (values).
+          Only the descriptions present will be mapped, others will be ignored.
+        - **list**: integer event codes (values) to include. Others will be
+          ignored. Event codes will be converted to string descriptions.
```

Comment:
This is not clear. I assume a list of strings is expected here, and event IDs are matched to list position, right?

Reply:
No it's a list of integers, containing the events IDs to select. 

I can remove if unclear.

Reply:
Maybe just clarify and don’t remove. I think you should accept an array too here.

Reply:
WDYT now?

Reply:
Yes, it's clearer now.